sdk/src/instruction.rs

```diff
@@ -96,11 +96,25 @@ pub struct AccountMeta {
     pub pubkey: Pubkey,
     /// True if an Instruciton requires a Transaction signature matching `pubkey`.
     pub is_signer: bool,
+    /// True if the `pubkey` can be loaded as a credit-only account.
+    pub is_credit_only: bool,
```

Comment:
This doesn't feel like a futureproof name. I'd rather think of attributes like this like file permissions. So an account may be executable and debitable, for example. How about is_debit? Or is_debitable?

Reply:
I like `is_debitable` more than `is_debit`.

Reply:
`is_debitable` works for me